### ISSUE TITLE
User Option in Entrypoint

### ISSUE THREAD

**Issue by stephancasas** [resyst-it/docker-bind9#7]
Is there any reason why the user needs to be specified in the entrypoint? I could not get this running in Docker Desktop without overriding, dropping the `-u` flag, and running named as root inside the container.

**Comment by resyst-it** [comment]
No, there's no specific reason. I just didn't want the process to run as root.

But I can switch and use USER and ARG.
Ex : 
```
ARG  user=named

USER ${user}
```
